phylo/src/substitution_models/dna_models/dna_substitution_parameters.rs

```diff
@@ -83,25 +89,19 @@ impl EvolutionaryModelParameters<DNAModelType> for DNASubstParams {
         }
     }
 
+    fn get_pi(&self) -> &FreqVector {
```

Comment:
btw, in Rust methods and fields can easily have the same name. For instance, this is a fully valid Rust code:
```rust
struct Hello {
    pub world: i32,
}

impl Hello {
    fn world(&self) -> i32 { self.world }
}

fn main() {
    let h = Hello{world: 123};
    println!("{}", h.world());
}
```

Reply:
Done.